templates/artist/artist_list.html

```diff
@@ -7,44 +7,34 @@
 {% endblock %}
 
 {% block content %}
-    <h1>Artists</h1>
+    <h2>Artists</h2>
     <div class="break">
         <hr>
     </div>
     <div class="list-nav">
         <div class="filter-sort-options center">
             <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
                 <li class="nav-float"><h6>Show me</h6></li>
-                <li class="is-dropdown-submenu-parent opens-right nav-float">
```

Comment:
Temporarily. After getting a lot of feedback from artists, they really wanted to be able to keep raising money continuously for a project. So when one campaign closes, another one opens up. 

The way this would work is: They want to give out a total of 20%. So they do the first round at 10%, the next one at 5%, then 2.5%, etc... 

The biggest reason for doing this is that bands have a hard time knowing how much they will need once the music is out. If it starts getting some traction, they want to raise more money to be able to promote. 

Another huge factor was trying to make it feel less like kickstarter. Crowdfunding has got a pretty bad wrap in the music world so when bands see the term "funded" it makes them run away. They really like the idea of the continuous investment. 

There could be a chance that an artist would no longer want to raise any money in the future, so I would hate to completely remove it and a situation come up in the future where we might need the feature. For now though, I would say it's okay to remove it if it's necessary.